runtime/tools/spec-builder/src/genesisBuilder.js

```diff
@@ -0,0 +1,51 @@
+const { Balances } = require("./balances-module");
+const { System } = require("./system-module");
+
+/**
+ * Class for the genesis configuration
+ */
+class GenesisBuilder{
+    genesis = {};
+
+    constructor(genesis){
+        this.genesis = genesis;
+    }
+    
+    /**
+     * Converts genesis property of the class to Raw
+     * @param genesis instance of class 
+     */
+    static toRaw(genBuilder) {
```

Comment:
the `genBuilder` actually is `genesisConfig` json file. So I think that it will be a better name for that parameter